<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Chatbot By Gemma</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="QFrame" name="frameMain">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QProgressBar" name="progress"/>
          </item>
          <item>
            <widget class="QFrame" name="frameSession">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                  <widget class="QPushButton" name="newSession">
                    <property name="text">
                     <string>New Session</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="SessionList" name="listSessions"/>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="QSplitter" name="splitter1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QFrame" name="frameView">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QWebEngineView" name="webView"/>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </widget>
    </item>
    <item>
      <widget class="QSplitter" name="splitter2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
      </widget>
    </item>
    <item>
      <widget class="QFrame" name="framePrompt">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
            <widget class="PromptEdit" name="prompt"/>
          </item>
          <item>
           <widget class="QPushButton" name="image">
            <property name="text">
             <string>Load Image</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="send">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
        </layout>
      </widget>
    </item>
  </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSaveAs"/>
   </widget>
   <addaction name="menu_File"/>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menu_Setting"/>
   <widget class="QMenu" name="menu_Functions">
    <property name="title">
     <string>Fun&amp;ctions</string>
    </property>
    <addaction name="actionParseFile"/>
    <addaction name="actionParseFunction"/>
   </widget>
   <addaction name="menu_Functions"/>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
    <action name="actionParseFile">
     <property name="text">
      <string>Parse &amp;File</string>
     </property>
    </action>
    <action name="actionParseFunction">
     <property name="text">
      <string>Parse Func&amp;tion</string>
     </property>
    </action>
    <action name="actionSetting">
     <property name="text">
      <string>&amp;Setting</string>
     </property>
    </action>
    <action name="actionSaveAs">
     <property name="text">
      <string>&amp;Save As ...</string>
     </property>
    </action>
    <action name="actionAbout">
     <property name="text">
      <string>&amp;About</string>
     </property>
    </action>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
